### PR TITLE
Kitware.ParaView version 5.10

### PR DESCRIPTION
**Pull request by G0rocks**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46571)

### CODE REVIEW

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
+PackageLocale: en-US
+Publisher: Kitware
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.locale.en-US.yaml:6]:
```suggestion
Publisher: Kitware, Inc.
```

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
```

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml:4]:
```suggestion
PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
```

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.yaml:4]:
```suggestion
PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
```

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.paraview.org/paraview-downloads/download.php?submit=Download&version=v5.10&type=binary&os=Windows&downloadFile=ParaView-5.10.0-Windows-Python3.9-msvc2017-AMD64.exe
+  InstallerSha256: 9C3655420C8E11D0DE14520A95FE3DCC0C295492B669A040CA83197F4A9A4951
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml:9]:
```suggestion
  InstallerSha256: 9C3655420C8E11D0DE14520A95FE3DCC0C295492B669A040CA83197F4A9A4951
  ProductCode: "ParaView 5.10.0-Windows-Python3.9-msvc2017-AMD64"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10/Kitware.ParaView.installer.yaml:7]:
```suggestion
  InstallerType: nullsoft
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:4]:
```suggestion
PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml (hunk 7 above)
Comment (G0rocks) [reply manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml]: I changed this to your suggestion (thanks btw!)

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageName: ParaView
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:3]:
```suggestion
```
This isn't needed for multi-file manifests.

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+PackageIdentifier: Kitware.ParaView
+Publisher: Kitware
+PackageLocale: en-US
+PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
+License: https://www.paraview.org/paraview-license/
+ShortDescription: ParaView is an open-source, multi-platform data analysis and visualization application
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:12]:
```suggestion
```
This isn't needed for multi-file manifests.

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageName: ParaView
+PackageIdentifier: Kitware.ParaView
+Publisher: Kitware
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:5]:
```suggestion
```
This isn't needed for multi-file manifests.

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+PackageLocale: en-US
+PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
+License: https://www.paraview.org/paraview-license/
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:11]:
```suggestion
```
This isn't needed for multi-file manifests.

File: manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml [hunk 12]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageName: ParaView
+PackageIdentifier: Kitware.ParaView
+Publisher: Kitware
+PackageLocale: en-US
```
Comment (ItzLevvie) [inline manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml:6]:
```suggestion
```
This isn't needed for multi-file manifests.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**